frontend/javascripts/admin/statistic/open_tasks_report_view.tsx

```diff
@@ -75,21 +93,36 @@ class OpenTasksReportView extends React.PureComponent<{}, State> {
               width={200}
             />
             <Column
-              title="# Assignments"
+              title="# Potentially Available Tasks"
```

Comment:
```suggestion
              title="# Available Tasks"
```

(I’d drop the “potentially” since this is what we are trying to say with the word “available”. Combined with the new text below, I’d say that is enough)